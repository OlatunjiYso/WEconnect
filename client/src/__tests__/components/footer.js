import React from "react";

import Footer from '../../components/footer';

describe('All tests for footer component', () => {
  
    it('should be defined', () => {
      expect(Footer).toBeDefined();
    });
    it('should render the footer', () => {
      const { wrapper } = shallow(<Footer />);
      expect(wrapper).toMatchSnapshot();
    });
});
