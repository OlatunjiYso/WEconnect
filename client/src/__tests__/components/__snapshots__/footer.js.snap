// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for footer component should render the footer 1`] = `undefined`;
